src/services/refact/diffs.ts

```diff
@@ -78,8 +81,11 @@ export const diffApi = createApi({
           body: { chunks: args.chunks, apply: args.toApply },
         });
 
-        if (result.error) {
-          return { error: result.error };
```

Comment:
Error could be a range of things, like failed requests... was there a type issue with `queryFn`?